### ISSUE TITLE
Docs Site Build broken following Renovate Version Upgrade Bumps

### ISSUE THREAD

**Issue by vorburger** [enola-dev/enola#367]
I just merged a bunch of [Renovate Version Upgrade Bumps](https://github.com/enola-dev/enola/pulls?q=is%3Apr+author%3Aapp%2Frenovate+is%3Aclosed).

One of them broke the Docs Site Build, note https://github.com/enola-dev/enola/actions/runs/7281554330 failed with:

```
Run actions/deploy-pages@v4
Fetching artifact metadata for github-pages in run 7281554330
Creating Pages deployment with payload:
{
	"artifact_id": 1127717[9](https://github.com/enola-dev/enola/actions/runs/7281554330/job/19842448122#step:2:10)56,
	"pages_build_version": "b780f2c2ff004379b58a8b2c8e4afcb744746702",
	"oidc_token": "***"
}
Error: Creating Pages deployment failed
Error: HttpError: No artifact_url provided
    at /home/runner/work/_actions/actions/deploy-pages/v4/node_modules/@octokit/request/dist-node/index.js:[12](https://github.com/enola-dev/enola/actions/runs/7281554330/job/19842448122#step:2:13)4:1
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at createPagesDeployment (/home/runner/work/_actions/actions/deploy-pages/v4/src/internal/api-client.js:68:1)
    at Deployment.create (/home/runner/work/_actions/actions/deploy-pages/v4/src/internal/deployment.js:78:1)
    at main (/home/runner/work/_actions/actions/deploy-pages/v4/src/index.js:30:1)
Error: Error: Failed to create deployment (status: 400) with build version b780f2c2ff004379b58a8b2c8e4afcb744746702. Responded with: No artifact_url provided
```

**Comment by vorburger** [comment]
> I just merged a bunch of [Renovate Version Upgrade Bumps](https://github.com/enola-dev/enola/pulls?q=is%3Apr+author%3Aapp%2Frenovate+is%3Aclosed).

The culprit is probably #361, which upgraded https://github.com/actions/deploy-pages from v2 to v4.

That v4 was only release just _yesterday._ I'll have a quick glimpse to see if it's easy to fix, otherwise just revert #361.

**Comment by vorburger** [comment]
Oh, wow; see https://github.com/actions/deploy-pages/issues/284 and https://github.com/actions/deploy-pages/issues/286 and https://github.com/actions/deploy-pages/issues/285 and https://github.com/actions/deploy-pages/issues/235.

I'm just going to revert BOTH #361 and #362, based [on this comment](https://github.com/actions/deploy-pages/issues/284#issuecomment-1864973669).

**Comment by vorburger** [comment]
Voilà, https://github.com/enola-dev/enola/actions/runs/7281741767 is 💚 green again.

**Comment by vorburger** [comment]
I've merged #370 and #371 now, and https://github.com/enola-dev/enola/actions/runs/7439916900 is green; all good.
